overlord/snapstate/snapmgr.go

```diff
@@ -77,8 +82,19 @@ func Manager(s *state.State) (*SnapManager, error) {
 		runner:  runner,
 	}
 
+	// this handler does nothing
+	runner.AddHandler("nop", func(t *state.Task, _ *tomb.Tomb) error {
+		return nil
+	}, nil)
+
 	runner.AddHandler("download-snap", m.doDownloadSnap, nil)
-	runner.AddHandler("install-snap", m.doInstallLocalSnap, nil)
+	runner.AddHandler("check-snap", m.doCheckSnap, nil)
+	runner.AddHandler("mount-snap", m.doMountSnap, m.undoMountSnap)
+	runner.AddHandler("copy-snap-data", m.doCopySnapData, m.undoCopySnapData)
+	runner.AddHandler("generate-security", m.doGenerateSecurity, m.undoGenerateSecurity)
+	runner.AddHandler("generate-wrappers", m.doGenerateWrappers, m.undoGenerateWrappers)
+	runner.AddHandler("update-current-symlink", m.doUpdateCurrentSymlink, m.undoUpdateCurrentSymlink)
```

Comment:
As discussed on telegram, we will consolidate (mount-snap and check-snap into mount-snap) and also (generate-wrappers and update-current-symlink into make-current)